GREEDY/Controllers/ItemsController.cs

```diff
@@ -33,6 +33,29 @@ public HttpResponseMessage Get(int id)
     }
 
     [EnableCors(origins: "*", headers: "*", methods: "*")]
+    public class GetAllUserItemsController : ApiController
+    {
```

Comment:
The name of the class sounds weird.

Reply:
Imagine that it is GetAllUserItems. Does that sound good to you? (The controller part is neccesary to make HTTP requests reach it). So I think everything is good here